images/manageiq-base/container-assets/container_env

```diff
@@ -4,8 +4,45 @@
 
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 
-DATABASE_USERINFO=""
-[[ -n "$DATABASE_USER" && -n "$DATABASE_PASSWORD" ]] && DATABASE_USERINFO="$(ruby --disable-gems -rcgi -e "puts \"#{CGI.escape(ENV['DATABASE_USER'])}:#{CGI.escape(ENV['DATABASE_PASSWORD'])}@\"")"
-DATABASE_QUERY="encoding=utf8&pool=5&wait_timeout=5&sslmode=${DATABASE_SSL_MODE:-prefer}"
-[[ -f /.postgresql/root.crt ]] && DATABASE_QUERY="${DATABASE_QUERY}&sslrootcert=/.postgresql/root.crt"
-export DATABASE_URL="postgresql://${DATABASE_USERINFO}${DATABASE_HOSTNAME:-localhost}:${DATABASE_PORT:-5432}/${DATABASE_NAME:-db_unknown}?${DATABASE_QUERY}"
+if [[ ! -f /var/www/miq/vmdb/config/database.yml ]]; then
+  [[ -f /run/secrets/postgresql/POSTGRESQL_HOSTNAME ]] && database_hostname_file=$(cat /run/secrets/postgresql/POSTGRESQL_HOSTNAME)
+  [[ -f /run/secrets/postgresql/POSTGRESQL_PASSWORD ]] && database_password_file=$(cat /run/secrets/postgresql/POSTGRESQL_PASSWORD)
+  [[ -f /run/secrets/postgresql/POSTGRESQL_PORT ]] && database_port_file=$(cat /run/secrets/postgresql/POSTGRESQL_PORT)
+  [[ -f /run/secrets/postgresql/POSTGRESQL_USER ]] && database_username_file=$(cat /run/secrets/postgresql/POSTGRESQL_USER)
+  database_hostname=${DATABASE_HOSTNAME:-$database_hostname_file}
```

Comment:
For local use, we used to have defaults for hostname as localhost, port as 5432, and database as db_unknown.  I think we need to bring this over for that use case.

I won't let this one hold up merge either, since it's a pure dev-diagnostic use case.

Reply:
Not sure how this works with HA, but some cloud providers have all apps setup to just hit `localhost:5432`, and the os deals with the redirect / health.